app/services/sensors/sensor_service.rb

```diff
@@ -0,0 +1,39 @@
+module Sensors
+  class SensorService
+    def initialize sensor
+      @sensor = sensor
+    end
```

Comment:
```suggestion
    def initialize; end
```
@sensor này a k thấy dùng